tests/test_backend.py

```diff
@@ -323,13 +323,18 @@ def test_set_call_empty_pipeline(self):
 
         with patch.object(pipeline, "set") as mocked_set:
             self.cache.set(
-                key, value, client=pipeline,
+                key,
+                value,
+                client=pipeline,
```

Comment:
Removing the last comma will let Black reformat the statement to a single line rather than forcing this to wrap.

Reply:
I had raised an issue regarding failure of job https://github.com/jazzband/django-redis/issues/471#issuecomment-707047419 and I got the above from there. 
So any further suggestions in order to avoid this?